src/sentry/uptime/models.py

```diff
@@ -374,6 +404,11 @@ def create_detector_from_project_subscription(project_sub: ProjectUptimeSubscrip
             "environment": env,
             "mode": project_sub.mode,
         },
+        workflow_condition_group=condition_group,
+    )
+    DetectorState.objects.create(
+        detector=detector,
+        active=True,
```

Comment:
What is active for? The comment is just `# If the detector is currently active` lol

Reply:
`active` tells the detector it has a data condition that has evaluated as truthy, like it's been "activated"

I don't like the name, i was thinking of changing it to `triggered` 🤣 -- especially once we add the `status` field to a detector / workflow for deletion status.

any thoughts or recommendations on a name that would be clear for you?

Reply:
hmm, shouldn't this be reflected in detector state?